resources/language/resource.language.de_de/strings.po

```diff
@@ -325,7 +325,7 @@ msgstr "Audio- und Untertitel-Einstellungen beibehalten"
 
 msgctxt "#30083"
 msgid "Force the display of forced subtitles only with the audio language set"
-msgstr ""
+msgstr "Anzeige erzwungener Untertitel nur erzwingen wenn Audiosprache gesetzt ist"
```

Comment:
There is a comma missing after "erzwingen".

Reply:
Thanks. fixed!